README.md

```diff
@@ -138,8 +136,8 @@ value), `min (minimum value), `sum` (sum of all the samples) and `sum2` (sum of
 attribute values and `occur` for attributes values of type string. Combining
 the information provided by these aggregated methods with the number of samples, it is possible to calculate probabilistic
 values such as the average value, the variance as well as the standard deviation. It is a mandatory parameter.
-* <b>aggrPeriod</b>: Aggregation period or resolution. For the time being, a fixed resolution determines the range as
-well as the origin time format and the possible offsets. It is a mandatory parameter.
+* <b>aggrPeriod</b>: Aggregation period or resolution. A fixed resolution determines the the origin time format and the
```

Comment:
Nice catch!!! Thanks ;) Fixed in https://github.com/telefonicaid/IoT-STH/commit/547215be6c2cb0399437d96e792567133744f3ac